packages/utils/src/media/trimCanvas.ts

```diff
@@ -38,28 +38,34 @@ export function trimCanvas(canvas: ICanvas): { width: number; height: number; da
     const context = canvas.getContext('2d', {
         willReadFrequently: true,
     });
-    const imageData = context.getImageData(0, 0, width, height);
-    const data = imageData.data;
 
-    let top = 0;
-    let bottom = height - 1;
-    let left = 0;
-    let right = width - 1;
-
-    while (top < height && checkRow(data, width, top)) ++top;
-    if (top === height)
+    if (context !== null)
```

Comment:
I prefer throwing a TypeError when `canvas.getContext` returns `null` instead of return `undefined` silently.